cmake/CMakeLists.txt

```diff
@@ -764,6 +765,13 @@ if(onnxruntime_USE_OPENVINO)
 
 endif()
 
+if (onnxruntime_USE_VITISAI)
+  add_definitions(-DUSE_VITISAI=1)
```

Comment:
It may be a good idea to add check for unsupported platforms. 
e.g. 
if(WIN32)
  message(FATAL_ERROR "Vitis-AI execution provider is not supported on Windows.")